new_google_recaptcha.gemspec

```diff
@@ -7,7 +7,7 @@ require "new_google_recaptcha/version"
 Gem::Specification.new do |s|
   s.name        = "new_google_recaptcha"
   s.version     = NewGoogleRecaptcha::VERSION
-  s.authors     = ["Igor Kasyanchuk"]
+  s.authors     = ["Igor Kasyanchuk", "rubyconvict"]
```

Comment:
I'm 100% agree, you did huge work

also please add yourself into "Contrinutors" section in readme